sanity/schemas/components/social.js

```diff
@@ -0,0 +1,18 @@
+export default {
+  name: 'social',
+  type: 'object',
+  fields: [
+    {
+      name: 'href',
+      type: 'string',
+      title: 'Link',
+      validation: (Rule) => Rule.required(),
+    },
+    {
+      name: 'icon',
```

Comment:
icon shouldn't be editable by Sanity. Let change the URL only for the editor in Sanity